libsrc/plus4/ser/plus4-stdser.s

```diff
@@ -353,26 +353,27 @@ SER_IOCTL:
 ;
 
 SER_IRQ:
-        lda     ACIA_STATUS     ; Check ACIA status for receive interrupt
-        and     #$08
-        beq     @L9             ; Jump if no ACIA interrupt (carry still clear)
-        lda     ACIA_DATA       ; Get byte from ACIA
-        ldx     RecvFreeCnt     ; Check if we have free space left
-        beq     @L1             ; Jump if no space in receive buffer
-        ldy     RecvTail        ; Load buffer pointer
-        sta     RecvBuf,y       ; Store received byte in buffer
-        inc     RecvTail        ; Increment buffer pointer
-        dec     RecvFreeCnt     ; Decrement free space counter
-        cpx     #33             ; Check for buffer space low
-        bcc     @L1             ; Assert flow control if buffer space low
+        lda     ACIA_STATUS     ;(4) ;status ;check for byte received
+        and     #$08            ;(2)
+        beq     @L9             ;(2*)
+
+@L1:    lda     ACIA_DATA       ;(4)  data  ;get byte and put into receive buffer
+        ldy     RecvTail        ;(4)
+        ldx     RecvFreeCnt     ;(4)
+        beq     @L3             ;(2*) Jump if no space in receive buffer
+        sta     RecvBuf,y       ;(5)
+        inc     RecvTail        ;(6)
+        dec     RecvFreeCnt     ;(6)
+        cpx     #33             ;(2)  check for buffer space low
+        bcc     @L2             ;(2*)
         rts                     ; Return with carry set (interrupt handled)
 
 ; Assert flow control if buffer space too low
 
-@L1:    lda     RtsOff
-        sta     ACIA_CMD
-        sta     Stopped
-        sec                     ; Interrupt handled
+@L2:    lda     RtsOff          ;(3) assert flow control if buffer space too low
+        sta     ACIA_CMD        ;(4) command
+        sta     Stopped         ;(3)
```

Comment:
Please always have a blank between the `;` and the first char of the comment. Please start comments with uppercase.